mw-webapp/src/component/editableTextarea/editableTextarea.css

```diff
@@ -0,0 +1,27 @@
+.grow-wrap {
```

Comment:
we don't use kebab-case in our app ) Please rename to camelCase

Reply:
done